moto/elbv2/models.py

```diff
@@ -293,11 +293,32 @@ def create_from_cloudformation_json(cls, resource_name, cloudformation_json, reg
         return load_balancer
 
     def get_cfn_attribute(self, attribute_name):
-        attributes = {
-            'DNSName': self.dns_name,
-            'LoadBalancerName': self.name,
-        }
-        return attributes[attribute_name]
+        '''
+        Implemented attributes:
+        * DNSName
+        * LoadBalancerName
+
+        Not implemented:
+        * CanonicalHostedZoneID
+        * LoadBalancerFullName
+        * SecurityGroups
+
+        This method is similar to models.py:FakeLoadBalancer.get_cfn_attribute()
+        '''
+        from moto.cloudformation.exceptions import UnformattedGetAttTemplateException
```

Comment:
Is there a reason this cant to at the top of the file?

Reply:
@terrycain: This is to work around a circular import (`moto.cloudformation` imports `elbv2.models`). The `elb.models.FakeLoadBalancer` class does the same thing, by the way (I mimicked it).

Reply:
Ahh I see, thats fine then.